pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -87,15 +102,12 @@ func (o *OssmInstaller) processManifests() error {
 }
 
 func (o *OssmInstaller) loadManifestsFrom(paths ...string) error {
-	manifestRepo, ok := o.GetRepoCache(kftypesv3.ManifestsRepoName)
-	if !ok {
-		return internalError(errors.New("manifests repo is not defined."))
-	}
-
 	var err error
 	var manifests []manifest
-	for i := range paths {
-		manifests, err = loadManifestsFrom(manifests, path.Join(manifestRepo.LocalPath, TMPL_LOCAL_PATH, paths[i]))
+	var manifestRepo = embeddedFiles
+	var rootDir = filepath.Join(baseOutputDir, o.Namespace, o.Name)
+	for _, p := range paths {
+		manifests, err = loadManifestsFrom(manifestRepo, manifests, p, rootDir)
```

Comment:
If we copy first, we don't need to deal with `var manifestRepo = embeddedFiles`, we can simply use the target path to where we copied files - which effectively can preserve old `loadManifestsFrom`.

Reply:
Perhaps even leading to removal of `templateDir` from `manifest` struct. Can you try to clean it up further?

Reply:
Yes, was able to remove `templateDir` from the manifest struct by passing the paths to the copied files :+1: 

[22c5bd0](https://github.com/maistra/opendatahub-operator/pull/5/commits/22c5bd064ae796de82cc573fed39882477baab56)